R/create_test_player.R

```diff
@@ -0,0 +1,6 @@
+#' Create a test player
+#' @export
+create_test_player <- function() {
+  player <- create_player()
+  return(player)
```

Comment:
Note: there is no need to add `return` in the last line, `return` is only needed for early returns. Still, it's fine :+1: